src/modules/CC1101/CC1101.h

```diff
@@ -499,6 +499,16 @@
 #define RADIOLIB_CC1101_GDO2_ACTIVE                            0b00000100  //  2     2     GDO2 is active/asserted
 #define RADIOLIB_CC1101_GDO0_ACTIVE                            0b00000001  //  0     0     GDO0 is active/asserted
 
+//Defaults
```

Comment:
The defaults have changed in 5.0.0 release, check the wiki: https://github.com/jgromes/RadioLib/wiki/Default-configuration#overview

Basically, all FSK modules are now set up for 4k8 baud by default.

Reply:
Changed here and in the `begin()`.